src/components/edit/trackedEntity/TrackedEntityDialog.js

```diff
@@ -110,6 +116,16 @@ class TrackedEntityDialog extends Component {
                 showRelationshipsChecked: true,
             })
         }
+
+        // Set org unit tree roots as default
+        if (!rows && orgUnits.roots) {
+            if (orgUnits.roots) {
```

Comment:
Is this check necessary? It's already checked on line 121.

Reply:
My bad - fixed!